packages/ts/generator-typescript-plugin-client/test/basic/fixtures/client.snap.ts

```diff
@@ -0,0 +1,3 @@
+import { ConnectClient as ConnectClient_1 } from "@vaadin/flow-frontend/Connect";
```

Comment:
should be `@vaadin/fusion-frontend`

Reply:
What's the conclusion here? This was marked as resolved but it's still saying `@vaadin/flow-frontend` there. Also same for the next resolved item below.

Reply:
it should be fusion-frontend, flow-frontend was a typo. There was another issue I found and Vlad is fixing them both at the same time.

Reply:
oops, fixed